packages/@lightningjs/ui-components/src/components/Button/ButtonSmall.d.ts

```diff
@@ -20,10 +20,7 @@ import lng from '@lightningjs/core';
 import Button, { ButtonStyle } from './Button';
 import { StylePartial } from '../../types/lui';
 
-type ButtonSmallStyle = ButtonStyle & {
-  prefixH: number;
-  suffixH: number;
-};
+type ButtonSmallStyle = ButtonStyle;
```

Comment:
we don't need to re-declare the type here if `ButtonSmall` has the same style type. we can remove this as well as the style accessors since they're the same type as `Button`'s

Reply:
Yeah something like this:

```
import lng from '@lightningjs/core';
import Button from './Button';

declare class ButtonSmall<
  TemplateSpec extends Button.TemplateSpec = Button.TemplateSpec,
  TypeConfig extends lng.Component.TypeConfig = lng.Component.TypeConfig
> extends Button<TemplateSpec, TypeConfig> {}

export default ButtonSmall;
```

Reply:
Looks like Control needed `ButtonSmallStyle`, so we'll need to either change it to use `ButtonStyle` or add `ButtonSmallStyle` back in if we want a middle type.